### PR TITLE
P 002 listings - Got Flask working with the DB and able to publish a list of animals using Jinja.

### PR DESCRIPTION
**Pull request by rokesby**

### CODE REVIEW

File: server/templates/listings.html
Comment (Matt-Wilkes) [inline server/templates/listings.html:1]: Can this one be kept out of main just to avoid any confusion?

File: server/templates/users.html
Comment (Matt-Wilkes) [inline server/templates/users.html:1]: Can this one be kept out of main just to avoid any confusion?

**Review by Matt-Wilkes — COMMENTED** [review]
submitted some feedback around the jinja templates/rendering

File: server/templates/users.html
Comment (rokesby) [reply server/templates/users.html:1]: Thanks @Matt-Wilkes  I'm using this for testing purposes. if you don't mind, can we keep in? I'm happy to rename on the next push to make it clear it's just for me. hope that works!

File: server/app.py [hunk 1]
```diff
@@ -1,35 +1,69 @@
...
-    # return render_template('emoji.html', emoji=':)')
+# Listings route - return a list of Animals.
+@app.route('/listings', methods=['GET'])
+def display_animals():
 
+    #with db_session() as db:
+    with app.app_context():
+        #animals = db.query(Animal).all()
+        animals = Animal.query.all()
+        return render_template('listings.html', animals=animals)
```
[the remark this answers is not in the document]
Comment (rokesby) [reply server/app.py:52]: @Matt-Wilkes  I'm intentionally rendering the template. For the react frontend, we might just create a duplicate method. I will most certainly retire my rendering template, but things are in a state of flux, so I want something for testing purposes. I can rename this to make it clear (sorry about that).

**Pull request merged**